pkg/util/cgroup/manager/v1/fs_linux.go

```diff
@@ -52,6 +52,14 @@ func (m *manager) ApplyMemory(absCgroupPath string, data *common.MemoryData) err
 		}
 	}
 
+	if data.TCPMemLimitInBytes > 0 {
```

Comment:
currently, multiple components will depend on those functions (`ApplyMemory`, `ApplyCPU` and so on), and we strongly relay on the potential appointments that different callers will only apply different fields, for instance, `TCPMemLimitInBytes` will be applied by `tcp-mem-handler` while  `LimitInBytes` will be applied by qrm sync loops. this potential appointments may have uncontrollable conflicts in the future, should we discuss and refine this? @csfldf

Reply:
In my previous understanding, the 'applyMemory' here is capable of configuring all parameters related to cgroupv1 memory, such as: kernel memory(memory.kmem.limit_in_bytes), memory charge(memory.move_charge_at_immigrate), oom ( memory.oom_control) and so on.

Reply:
@csfldf Any comments?

Reply:
I think this function is similar to [WithResources](https://github.com/containerd/containerd/blob/main/pkg/cri/opts/spec_linux_opts.go#L304). It's capable of configuring multi-parameters and validate input with zero-value judgement.